requirements.txt

```diff
@@ -12,10 +12,9 @@ PyYAML>=5.3.1
 requests>=2.23.0
 scipy>=1.4.1
 thop>=0.1.1  # FLOPs computation
-torch>=1.7.0  # see https://pytorch.org/get-started/locally (recommended)
-torchvision>=0.8.1
+torch==2.5.1  # see https://pytorch.org/get-started/locally (recommended)
```

Comment:
I assume this was done because torch 2.6.0+ has `weights_only` defaulting to True?

Instead of locking the torch version, it's better to add `weights_only=False` parameter to `torch.load()` calls, and bump minimal version to `torch>=2.0.0` (where this option was first introduced).

Or just set env variable `TORCH_FORCE_NO_WEIGHTS_ONLY_LOAD=yes` in CI -- this would work with any PyTorch release without bumping the min version.